dev-tools/packaging/templates/ironbank/auditbeat/hardening_manifest.yaml

```diff
@@ -14,7 +14,7 @@ tags:
 # Build args passed to Dockerfile ARGs
 args:
   BASE_IMAGE: "redhat/ubi/ubi9"
-  BASE_TAG: "9.6"
+  BASE_TAG: "\"9.7\""
```

Comment:
@v1v what are we doing with this? Fixing manually? Fixing the automation?

Reply:
https://github.com/elastic/beats/actions/runs/19359336122 , should now create a PR without those escaped quotes